lib/matplotlib/tests/test_quiver.py

```diff
@@ -133,6 +134,20 @@ def test_barbs():
              cmap='viridis')
 
 
+@cleanup
+@raises(ValueError)
```

Comment:
If we're allowed to explicitly use pytest now, I will absolutely do that. I just tried to be consistent with what's already around.